cmd/kubelet/app/plugins.go

```diff
@@ -65,6 +68,9 @@ func ProbeVolumePlugins(featureGate featuregate.FeatureGate) ([]volume.VolumePlu
 	allPlugins = append(allPlugins, projected.ProbeVolumePlugins()...)
 	allPlugins = append(allPlugins, local.ProbeVolumePlugins()...)
 	allPlugins = append(allPlugins, csi.ProbeVolumePlugins()...)
+	if utilfeature.DefaultFeatureGate.Enabled(features.ImageVolume) {
```

Comment:
shouldn't this be `featureGate.Enabled(...)` ? we have them passed in and they seem to be used to check volume plugins above

Reply:
Yes true, great catch! Fixed as suggested.